app/styles/_customization.scss

```diff
@@ -0,0 +1,3 @@
+.ember-basic-dropdown-content {
+  z-index: 1002;
```

Comment:
That would be nice if the ember-modal-dialog contains the ember-basic-dropdown. Unfortunately, the ember-modal-dialog does not contain the ember-basic-dropdown, that's why I think we make the z-index of ember-basic-dropdown globally